packages/core/src/envelope.ts

```diff
@@ -109,6 +114,15 @@ export function createSpanEnvelope(spans: SentrySpan[]): SpanEnvelope {
     sent_at: new Date().toISOString(),
     ...(dscHasRequiredProps(dsc) && { trace: dsc }),
   };
-  const items = spans.map(span => createSpanEnvelopeItem(spanToJSON(span)));
+
+  const beforeSend = client && client.getOptions().beforeSendSpan;
```

Comment:
```suggestion
  const beforeSendSpan = client && client.getOptions().beforeSendSpan;
```

let's call it that, I was confused for a sec because we also have an actual `beforeSend` hook (for error events) 😅

Reply:
Right, not sure why I named it that way 😅 ...